jest.config.js

```diff
@@ -35,6 +35,9 @@ module.exports = {
     prefix: "<rootDir>/",
   }),
 
+  globals: {
+    Uint8Array: Uint8Array,
+  },
```

Comment:
We need to update jest to 29 to avoid this. Esbuild check the environment before start. 
Jest manually copy global objects from VM environment to Jest Isolated Environment. And jest able to copy only those entries which were known on the moment of releasing jest. Esbuild needs  Uint8Array to work.